app/pages/household/[houseId]/index.tsx

```diff
@@ -112,23 +184,102 @@ const Household = () => {
-              </h3>
-              <p className="py-4">
-                This action will delete the house for all users.
-              </p>
-              <input
-                className="btn"
-                onClick={handleLeaveHouse}
-                value="leave house instead"
-              ></input>
+              <h3 className="text-lg font-bold">Household Settings</h3>
+              {!isOwner() && (
+                <div className="flex flex-col mt-5">
+                  <h3 className="text-xl font-bold">Leave House</h3>
+                  <p>
+                    You&apos;ll need a new invitation to join this house again!
+                  </p>
+                  <input
+                    className="btn btn-error rounded-lg place-self-center mt-5"
+                    onClick={handleLeaveHouse}
+                    value="leave house"
+                  ></input>
+                </div>
+              )}
               <br></br>
-              <input
-                className="btn"
-                onClick={handleDeleteHouse}
-                value="delete house"
-              ></input>
+              {isOwner() && (
+                <div className="flex flex-col">
+                  <form action="#" onSubmit={handleDelegateOwner}>
+                    <h3 className="text-xl font-bold">Delegate Ownership</h3>
+                    <p>
+                      Choose someone to handover the ownership of this UHouse!
+                    </p>
+                    <br />
+                    <div className="flex gap-x-5">
+                      {householdMembers && householdMembers.length > 0 ? (
+                        <>
+                          <select
+                            name="newOwner"
+                            id="newOwner"
+                            className="select select-primary w-full max-w-xs"
+                          >
+                            <option disabled selected>
+                              Select the new owner!
+                            </option>
+                            {householdMembers.map((member: any) => {
+                              return (
+                                <option key={member[0]} value={member}>
+                                  {member[1]}
+                                </option>
+                              );
+                            })}
+                          </select>
+                          <input
+                            type="submit"
+                            className="btn btn-warning"
+                            value="Delegate"
+                          ></input>
+                        </>
+                      ) : (
+                        <>
+                          <select
+                            name="newOwner"
+                            id="newOwner"
+                            className="select w-full max-w-xs"
+                            disabled
```

Comment:
looks good but why are these disabled? Not able to choose someone to delegate to on preview